src/app/settings/payment.component.ts

```diff
@@ -67,7 +67,7 @@ export class PaymentComponent implements OnInit {
         this.stripeScript.src = 'https://js.stripe.com/v3/';
         this.stripeScript.async = true;
         this.stripeScript.onload = () => {
-            this.stripe = (window as any).Stripe(process.env.ENV === 'production' && !platformUtilsService.isDev()  ?
```

Comment:
Why was stripe removed?

Reply:
It wasn't, we just changed the ENV name.